uber/models.py

```diff
@@ -629,8 +629,18 @@ def everything(self, location=None):
                                         .options(joinedload(Attendee.shifts), joinedload(Attendee.group))
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
+
+            # PERFORMANCE OPTIMIZATION: Job.available_volunteers uses a @cached_property decorator.  This decorator
+            # populates job._available_volunteers.  Because this function is returning a huge amount of Jobs,
+            # we will pre-populate it here and skip the myriad database queries which would occur if we called
+            # .available_volunteers many times in a row.
+            #
+            # Note that this isn't exactly the same output as .available_volunteers, but this code should be kept
+            # in-sync as much as possible with Job.available_volunteers.  This is messy but needed to work around perf
+            # problems with pages that show large amounts of Jobs using everything()
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_volunteers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
+
```

Comment:
renamed available_staffers to available_volunteers, made it use the caching property, and added this comment here.  should all work as intended.

one important counter-intuitive here which I noted prominently in the comments:  _available_volunteers here is not going to return the exact, exact same result as calling .available_volunteers

I think from reading both of the code paths that it's damn close and the only difference is in the final ordering, and that this is an acceptable tradeoff for the perf benefits.  We should make a point sometime of measuring this optimization on a large dataset just to make sure it holds true.  For now, this is fine.